.github/ISSUE_TEMPLATE/CODEOWNERS

```diff
@@ -0,0 +1 @@
+*       @dotnet/roslyn-compiler
```

Comment:
This should likely just be the proposals folder, not the whole repo.

Reply:
I'd be open to that. But there's been a couple of accepted PRs (mostly typos) to the specs folder as well, which is useful to get notifications for.

Reply:
I don't think the whole compiler team needs to be notified for that though. Perhaps we should make a team with people who actually participate on csharplang for those types of changes? You, me, Mads, @BillWagner, and Jared?

Reply:
@dotnet/roslyn-compiler Please reply if you want to be notified as a reviewer for csharplang PRs.

Reply:
I think it's no big deal to notify everyone from the compiler team. If people aren't happy about the notifications we can revisit.